src/libraries/System.Diagnostics.DiagnosticSource/ref/System.Diagnostics.DiagnosticSourceActivity.cs

```diff
@@ -437,6 +437,7 @@ public sealed class InstrumentAdvice<T> where T : struct
         public Measurement(T value, System.Collections.Generic.IEnumerable<System.Collections.Generic.KeyValuePair<string, object?>>? tags) { throw null; }
         public Measurement(T value, params System.Collections.Generic.KeyValuePair<string, object?>[]? tags) { throw null; }
         public Measurement(T value, params System.ReadOnlySpan<System.Collections.Generic.KeyValuePair<string, object?>> tags) { throw null; }
+        public Measurement(T value, in TagList tags) { throw null; }
```

Comment:
```suggestion
        public Measurement(T value, in System.Diagnostics.TagList tags) { throw null; }
```
ref source always use full type name